tools/snp.sh

```diff
@@ -68,25 +68,26 @@ trap cleanup EXIT
 # Working directory setup
 WORKING_DIR="${WORKING_DIR:-$HOME/snp}"
 SETUP_WORKING_DIR="${SETUP_WORKING_DIR:-${WORKING_DIR}/setup}"
-LAUNCH_WORKING_DIR="${LAUNCH_WORKING_DIR:-${WORKING_DIR}/launch}"
+GUEST_NAME="${GUEST_NAME:-snp-guest}"
+LAUNCH_WORKING_DIR="${LAUNCH_WORKING_DIR:-${WORKING_DIR}/launch/${GUEST_NAME}}"
```

Comment:
LAUNCH_WORKING_DIR="${LAUNCH_WORKING_DIR:-${WORKING_DIR}/launch}"

Add command line option for separate user guest name
   --guest_name          Specify different guest name